linux_os/guide/system/network/network-ipv6/configuring_ipv6/disabling_ipv6_autoconfig/sysctl_net_ipv6_conf_all_accept_source_route.rule

```diff
@@ -2,7 +2,7 @@ documentation_complete: true
 
 prodtype: rhel7
 
-title: 'Configure Kernel Parameter for Accepting Source-Routed Packets for All Interfaces'
+title: 'Configure Kernel Parameter for Accepting Source-Routed Packets for All Interfaces via IPv6'
```

Comment:
`Configure Kernel Parameter for Accepting Source-Routed Packets for All IPv6 Interfaces`